scripts/count_hours.py

```diff
@@ -0,0 +1,59 @@
+import warnings
+import argparse
+import librosa
+SAMPLE_RATE = 16000
+
+
+def read_audio_and_text(inputs):
+    audio_path = inputs['file_path']
+    text = inputs['text']
+    language = inputs['language']
+    text = ' '.join(text.split())
+    for p in ',.:;?!-_':
+        text = text.replace(p, '')
+    text = text.lower().split()
+    audio, sr = librosa.load(audio_path, sr=SAMPLE_RATE, mono=True)
+    return audio.size / float(sr)
+
+
+def process_line(args, line):
+    filename, language, text = line.split(args.delimiter)
+    inputs = {
+        'file_path': filename,
+        'text': text.strip(),
+        'language': language
+    }
+    try:
+        return read_audio_and_text(inputs)
+    except Exception as err:
+        print(str(err))
```

Comment:
Print error and return is exactly the same as not catching the error. I'd just remove try-except block.

Reply:
It's done to prevent crash. This try-catch block would just skip faulty inputs